spec/features/budgets/investments_spec.rb

```diff
@@ -966,6 +966,24 @@ def investments_order
     expect(page).to have_content(investment.unfeasibility_explanation)
   end
 
+  scenario "Show (unfeasible budget investment with valuation not finished)" do
+    user = create(:user)
+    login_as(user)
+
+    investment = create(:budget_investment,
+                        :unfeasible,
+                        valuation_finished: false,
+                        budget: budget,
+                        group: group,
+                        heading: heading,
+                        unfeasibility_explanation: 'Local government is not competent in this matter')
+
+    visit budget_investment_path(budget_id: budget.id, id: investment.id)
```

Comment:
I order to make the budget links work right (related to issue https://github.com/AyuntamientoMadrid/consul/issues/1163) I started a refactor at https://github.com/consul/consul/pull/2463 to always avoid using id's in this kind of paths.

Reply:
That is part of a bigger refactoring, the above syntax is consistent with the test above